### DISCUSSION TITLE
4.10.0 issue: Anyone getting 403s attempting to upgrade to websocket connections?

### DISCUSSION THREAD

**Discussion by duckboy81** [miguelgrinberg/python-engineio#379] [General]
Hey there, just a possible PSA: we spent our entire day troubleshooting an issue with `chaintlit` where we were stuck with long-polling and any ugprades to websockets were immediately closed.

Didn't have time to track down the source of the issue, but rolling back to `4.9.1` fixed everything. If others are experiencing the same issue, it may be worth figuring out the root cause.

Thanks!

**Comment by miguelgrinberg** [comment]
Please upgrade to 4.10.1, there was a problem with 4.10.0 that I just addressed.
